.betterer.results

```diff
@@ -4834,10 +4834,6 @@ exports[`better eslint`] = {
       [0, 0, 0, "No untranslated strings in text props. Wrap text with <Trans /> or use t()", "3"],
       [0, 0, 0, "No untranslated strings in text props. Wrap text with <Trans /> or use t()", "4"]
     ],
-    "public/app/features/logs/components/LogRows.tsx:5381": [
```

Comment:
Not sure how these betterer changes became a part of this PR

Reply:
i've run betterer, which wasn't run on main i think, hence those changes. I've reverted.